### PR TITLE
Online Retail data set.

### PR DESCRIPTION
**Pull request by ajdapretnar**
Data set for clustering products.

### CODE REVIEW

**Comment by BlazZupan** [comment]
Please add the source, like for other UCI data files. An example entry is <a href='https://archive.ics.uci.edu/ml/datasets/Wine'>UCI ML Repository</a>

Also, add a link to the publisher for the paper (anchor the title of the paper, the link is at https://link.springer.com/article/10.1057/dbm.2012.17).

There is a space missing before one of the "." in description.

**Comment by ajdapretnar** [comment]
Sure, I thought I did, but I obviously forgot. Will add it! Also I will add 'see also' once I finish the blog.

File: core/retail-basket.tab.info [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "version": "1.0",
+    "collection": "UCI",
+    "instances": 914,
+    "missing": false,
+    "variables": 50,
+    "year": "2012",
+    "references": [
+        "Daqing Chen, Sai Liang Sain, and Kun Guo (2012) Data mining for the online retail industry: A case study of RFM model-based customer segmentation using data mining. Journal of Database Marketing and Customer Strategy Management 19(3), pp. 197."
+    ],
+    "size": 665,
```
Comment (lanzagar) [inline core/retail-basket.tab.info]: Not sure what this is, but it is not the size of the file in bytes.
Please delete this field - it is added automatically later.

**Comment by ajdapretnar** [comment]
It should be fixed now, please see and merge.

File: core/retail-basket.tab.info [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+    "name": "retail-basket",
+    "description": "This is a transnational data set which contains all the transactions occurring between 01/12/2010 and 09/12/2011 for a UK-based and registered non-store online retail. The company mainly sells unique all-occasion gifts. Many customers of the company are wholesalers.",
```
Comment (lanzagar) [inline core/retail-basket.tab.info]: Is that 1st of Dec to 9th of Dec or Jan 12th to Sep 12th. It is a UK data set so maybe the first, but US users will see it differently.
I vote for the scientific version of using the ISO format!

File: core/retail-basket.tab.info [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    "title": "Online Retail",
+    "tags": ["marketing"],
+    "target": "",
```
Comment (lanzagar) [inline core/retail-basket.tab.info]: `"target": null,`
(cf. README)

File: core/retail-basket.tab.info [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    "target": null,
+    "version": "1.0",
+    "collection": "UCI",
+    "instances": 914,
+    "missing": false,
+    "variables": 50,
+    "year": "2012",
+    "references": [
+        "Daqing Chen, Sai Liang Sain, and Kun Guo (2012) <a href='https://link.springer.com/article/10.1057/dbm.2012.17'>Data mining for the online retail industry: A case study of RFM model-based customer segmentation using data mining.</a> Journal of Database Marketing and Customer Strategy Management 19(3), pp. 197."],
+    "source": "<a href:'http://archive.ics.uci.edu/ml/datasets/online+retail'>http://archive.ics.uci.edu/ml/datasets/online+retail</a>",
```
Comment (lanzagar) [inline core/retail-basket.tab.info]: I missed this:
<a href:    ->   <a href=
Also, to bo consistent with other UCI data sets, make the source say "UCI ML Repository"
`"source": "<a href='http://...'>UCI ML Repository</a>",`

**Pull request merged**